var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -330,48 +463,80 @@ def post_configure_fixes(self):
                         '.tf_configure.bazelrc')
 
     def build(self, spec, prefix):
+        # https://docs.bazel.build/versions/master/command-line-reference.html
+        args = [
+            # Don't allow user or system .bazelrc to override build settings
+            '--ignore_all_rc_files',
+            'build',
+            # Spack logs don't handle colored output well
+            '--color=no',
+            '--jobs={0}'.format(make_jobs),
+            '--config=opt',
+        ]
+
+        if '+mkl' in spec:
+            args.append('--config=mkl')
+        else:
+            args.append('--config=nomkl')
```

Comment:
do we want to use a helper function for this? This reads like the bazel-version of `with-or-without`

Reply:
Let me test things to make sure this syntax actually works first.